modules/ding_smartbanner/ding_smartbanner.install

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Install and uninstall ding_smartbanner.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function ding_smartbanner_install() {
+  $smartbanner_config = array(
```

Comment:
By setting all this default values during installation. One could omit all the "empty" checks in the administration form as they always will exists.

Reply:
"Empty" check should be present because there are non-mandatory fields. In fact mandatory fields are the one which are described in 'base_config' section of config array, the rest of settings are allowed to be empty.

Reply:
👍